python/pyspark/sql/functions.py

```diff
@@ -275,15 +273,6 @@ def _():
 del _name, _doc
 
 
-@since(1.3)
-def approxCountDistinct(col, rsd=None):
```

Comment:
Looks like the removal of this is causing the test failure, maybe do a grep for `approxCountDistinct` in the tests?